packages/elements-core/src/components/Docs/HttpOperation/Badges.tsx

```diff
@@ -52,7 +36,7 @@ export const SecurityBadge: React.FC<{ scheme: HttpSecurityScheme; httpServiceUr
   httpServiceUri,
 }) => {
   const badge = (
-    <Badge icon={faLock} className="sl-bg-gray-6 sl-max-w-xs sl-truncate">
+    <Badge icon={faLock} role="badge" className="sl-truncate" style={{ backgroundColor: badgeDefaultColor }}>
```

Comment:
`badge` is not a valid role. [See spec](https://www.w3.org/TR/wai-aria-1.1/#roles). I know it has been here, I also mentioned it before, but now that you are changing tests anyway we could clean this up too.